modules/persistent-ebs/locals.tf

```diff
@@ -0,0 +1,11 @@
+locals {
+  volume_default = {
+    type        = "gp2"
+    iops        = 0
+    size        = 15
+    encrypted   = true
+    kms_key_id  = ""
+    snapshot_id = ""
+  }
+  volumes_default = [for x in var.volumes : merge(local.volume_default, x)]
```

Comment:
Does this mean the module _always_ uses the default? I don't see how that's optional.

Reply:
No, this merges user value and default. Without this, user must specify all values of the map, of each element (EBS volume) of the list. Now he just needs to specify the necessary ones.